src/components/minting/FeaturedArtists.tsx

```diff
@@ -18,36 +19,54 @@ interface FeaturedArtistsProps {
 const FeaturedArtists: FunctionComponent<FeaturedArtistsProps> = ({
   featured,
   onDelete,
+  isDeleteDisabled = false,
 }) => {
   const theme = useTheme();
 
   return (
     <Box>
-      { featured.map((featuredArtist) => (
-        <Stack
-          key={ featuredArtist.email }
-          sx={ {
-            flexDirection: "row",
-            justifyContent: "space-between",
-            alignItems: "center",
-            mt: 2,
-          } }
-        >
-          <Typography variant="subtitle1">{ featuredArtist.email }</Typography>
+      { featured.map((featuredArtist) => {
+        const statusContent = getCollaboratorStatusContent(
+          featuredArtist.status
+        );
 
-          <Button
-            color="white"
-            sx={ { ml: 3 } }
-            variant="secondary"
-            width="icon"
-            onClick={ () => {
-              onDelete(featuredArtist, featured);
+        return (
+          <Stack
+            key={ featuredArtist.email }
+            sx={ {
+              flexDirection: "row",
+              justifyContent: "space-between",
+              alignItems: "center",
+              mt: 2,
             } }
           >
-            <CloseIcon sx={ { color: theme.colors.white } } />
-          </Button>
-        </Stack>
-      )) }
+            <Stack direction="row" gap={ 1 } alignItems="center">
+              { statusContent && (
+                <Tooltip title={ statusContent.tooltip }>
+                  { statusContent.icon }
+                </Tooltip>
+              ) }
+
+              <Typography variant="subtitle1">
+                { featuredArtist.email }
+              </Typography>
+            </Stack>
+
+            <Button
```

Comment:
in `src/pages/home/library/EditSong.tsx` we are displaying the delete icon if `!(isMinted || isPending)`. here we are disabling it but still showing it. I would think we want to do the same for both cases, im fine either way as long as we are consistent